pkg/operation/botanist/coredns.go

```diff
@@ -43,11 +43,11 @@ func (b *Botanist) DefaultCoreDNS() (coredns.Interface, error) {
 	values := coredns.Values{
 		// resolve conformance test issue (https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/dns.go#L44)
 		// before changing
-		ClusterDomain:                   gardencorev1beta1.DefaultDomain,
-		ClusterIP:                       b.Shoot.Networks.CoreDNS.String(),
-		Image:                           image.String(),
-		PodNetworkCIDR:                  b.Shoot.Networks.Pods.String(),
-		NodeNetworkCIDR:                 b.Shoot.GetInfo().Spec.Networking.Nodes,
```

Comment:
was this previously prone to `nil` panics?

Reply:
Potentially. Although it might be impossible to get to this point if shoot.Spec.Network is nil. The shoot object also sets the `Networking` field to an empty object (in `SetDefaults_Shoot`).